---
title: "Journal Club Outline"
author: "Matthiew Haines"
output: 
  powerpoint_presentation:
      slide_level: 2
     # reference_doc: 2022-CFAES-PPT-Template-2.pptx
bibliography: MyLibrary.bib
---

## Outline:
1.  Review some basic taste biology and genetics  
2.  Review and critique @Hwang2018  

# PROP as a case study  

## Discovering PROP  
:::::::::::::: {.columns}
::: {.column}
-   Variability in bitter taste perception first described in 1932^1^
-   Arthur Fox reported inter-subject variability in PTC perception
    -   discrete categories of "taster" (sensitive) and "non-taster" (insensitive)
-   PTC is toxic (not great for human science), so PROP has widely replaced PTC
    -   PROP acts in a similar manner to PTC
-   Both PROP and PTC are agonists of *TAS2R38*

:::
::: {.column}
![@Behrens2013](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/PROP-PTC.jpeg)

:::
::::::::::::::

## Biology Review  
:::::::::::::: {.columns}
::: {.column}
- "Taster status" of PROP is the result of your **phenotype**^2^
  + the expression of a certain trait
- Phenotype is the result of your **genotype**  
  + the genetic make up or specific combination of alleles
  + alleles are one of the versions of gene  

:::

::: {.column}
![[National Human Genome Research Institute](https://www.genome.gov/genetics-glossary/genotype)](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/Genotype.jpg)

:::
::::::::::::::


  
## Back to PROP
:::::::::::::: {.columns}
::: {.column}
![@Genick2011](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/Genick2011-Fig4.png)

:::
::: {.column}
- rs10246939 (on chromosome 7)
  + explains ~50% of variance seen in detection thresholds^1^
  + this SNP here has a MAF of ~46%^1^
- This is at a **SNP**, or single nucleotide polymorphism
  + a SNP is a single change in a base pair, resulting in genetic variation
  + usually high frequency^3^
  + *mutations* are generally reserved for low frequency genetic variation^3^

:::
::::::::::::::
::: notes
- can see the alleles here are **T** and **C**    
:::

## Other factors influencing PROP sensitivity   
From @Robino2022:  

  -   fungiform pappilae count/density
  -   age
  -   gender
  -   salivary flow rate and protein composition
  
# @Hwang2018

## Introduction  
-   Bitterness is a basic taste, but isn't perceived the same way for everyone
    -   36-73% of bitterness perception is due to genetics^6^
-   Other factors can influence taste perception^4^  
-   PROP bitterness has been implicated in fat perception^7^
-   PTC related to diseases^8^, immune system, and surgical outcome of severe rhinosinusitis^6^
-   Want to understand "underlying genetic variation that leads to individual differences in bitter perception"
    -   this could have implications in "personalized medicine"^3,6^  

## Previous GWAS study - @Reed2010
:::::::::::::: {.columns}
::: {.column}
![@Reed2010](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/ReedFig1.jpeg)

:::
::: {.column}
- Previous GWAS^5^ replicated association between *TAS2R38* and PROP perception
- Identified a SNP within *TAS2R19*: accounting for 5.8% of variance in quinine perception
- Did not find loci for caffeine, SOA, or proposed second locus for PTC/PROP
  
:::
::::::::::::::

::: notes
other suggested/proposed signals for PTC/PROP on chromosomes 5 and 16  
:::

## The New Approach
-   Use **multivariate association** analysis to find common genetic variants (MAF $\ge$ 5%) with small effects
-   Can identify **pleiotropic** genetic variants
    + one gene controls expression of multiple/several phenotypic traits
    + ex. sickle cell disease^9^
-   Using MV GWAS can find associations that didn't reach significance in univariate analysis
    + does not change effect size^6^

##
:::::::::::::: {.columns}
::: {.column}
![@Hwang2018](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12864-018-5058-2/MediaObjects/12864_2018_5058_Fig1_HTML.png?as=webp)
:::
:::{.column}
-   A: spread on y axis, not on x axis
    -   only T1 (y axis) is associated with variant (non-pleiotropic)
    -   better separation seen in 2D 
-   B: spread on y and x axis
    -   minor allele has opposing effects, $\uparrow$ T1 (Y) and $\downarrow$ T2 (X)
    -   starting to see the spread on y and x axes, especially when combined
-   C: minor allele has similar effects, $\uparrow$ T1 (Y) and $\uparrow$ T2 (X)
    -   no greater separation with bivariate analysis than would get with univariate

:::
::::::::::::::
::: notes
practice with the pencil test from high school math class
:::

# Methods
## What is a GWAS?
- GWAS studies aim to identify genetic risk factors for common diseases by utilizing DNA sequence variations^3^  
  + the goal/outcome is to identify biological underpinnings of disease susceptibility, predict at-risk groups, and develop new prevention/treatments
- Tests each SNP for an independent association to a phenotype
- Is not necessarily hypothesis driven
  + *replications might be*

## How is GWAS used in @Hwang2018

:::::::::::::: {.columns}
::: {.column}
![@Bush2012](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e2a/3531285/2e868bea85f6/pcbi.1002822.g001.jpg)  
:::
:::{.column}
-   Identify common genetic variants with small effect sizes (lower right of figure)
    -   these variants may have been underpowered in the previous GWAS study^5,10^  
-   1999 Caucasians from Europe, including twins and siblings
    -   this is an increase in sample size from @Hansen2006 by 40%
-   Used a Bonferroni corrected p \< $1*10^{-8}$
    
:::
::::::::::::::
::: notes
- Lower Right - GWAS  
- Upper Right - Linkage analysis and GWAS
- Lower Left - difficult to challenge (not appropriate for GWAS)
- *NOD2 was identified by linkage analysis, not GWAS* (@Verstockt2018)

::: 

## Modeling

-   Linear mixed model
    -   covariates (fixed effects): age, sex, ear infection history, first 5 PC from genotype
    -   used additive model: for each copy of allele *A*, there is a uniform, linear increase in risk^3^
-   Bivariate analysis can complement univariate analysis
    -   can enhance the strength of a SNP association, but not effect size^6^

::: notes
Variance explained by SNP was calculated as $2* MAF* (1-MAF)* \beta^2$
:::


## Taste Perception
-   Presented 5 bitter samples with reps
    -   0.181 mM quinine HCl
    -   0.050 M caffeine
    -   0.6 mM PROP
    -   0.2 mM SOA
    -   0.00499 mM denatonium benzoate
    -   same order presentation for all participants
-   Previous GWAS^5^ used 1 mL solution
-   Swish and spit after 5 s, rate on LMS^10^
-   Bitter scores were square root transformed then converted to z-scores

::: notes
- 1 mL is not really enough to get a perception of taste, it's like how much spit you can save up before you swallow
- spitting may present difficulty in regulating how participants perceived taste (if not swishing vigorously across the whole mouth)
- this is the dependent variable, so limitations here can *really* hinder the data quality and interpretation
- $4.99 * 10^{-6}$ M DB
:::

## Functional annotation of identified SNPs

-   Functional annotation can be used to identify any potential roles of the identified SNPs
    -   could be used to identify/develop mechanistic hypotheses in understanding variance in taste perception based on the identified/associates SNPs

::: notes
*do I need to expand this section?*  
:::

# Results
::: notes
**Big takeaways**
- confirmed associations with PROP (rs10246939) and quinine (rs10772420) from @Reed2010
- identified and provide independent rep for caffeine (rs2597979)
::: 

## PROP 
:::::::::::::: {.columns}
:::{.column}
- Strong association with rs10246939 on chromosome 7 (the "classic" SNP)
  + seen in @Reed2010 as well
- No signals on chromosomes 5 or 16, where additional loci are proposed
- PROP paper showed association on chromosome 2, rs6761655
  + appeared in @Reed2010 but was weak and therefore not reported
  + not reported in other GWAS studies using PROP
    
:::
::: {.column}
![@Hwang2018](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/FigS2.png)
:::
::::::::::::::

## Quinine
:::::::::::::: {.columns}
:::{.column}
- rs10772420 leads to missense variant on TAS2R19 (chromosome 12)
  - Explained the same amount of variance from previous study^5^ (5.7%)
- Quinine-rs10772420 association remained significant after conditional analysis with rs2597979 (top caffeine SNP) and rs67487380 (top SOA SNP)
- Caffeine and quinine were analyzed in bivariate analysis
  - small association (p = 0.0025, NS)
  - minor allele showed decrease in caffeine and increase in quinine perception
:::
::: {.column}
![@Hwang2018](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12864-018-5058-2/MediaObjects/12864_2018_5058_Fig2_HTML.png) 

:::
::::::::::::::

::: notes
  - quinine-rs2597979 association (p = 0.044) seen
    + could this suggest rs2597979 has a pleiotropic effect?
- similar results seen with SOA and DB (@Delwiche2001)
  - minor allele in same direction as quinine
- quinine-caffeine results in greater separation (Fig 1b)
:::

## Caffeine  
:::::::::::::: {.columns}
:::{.column}
- Peak at rs2597979 (chromosome 12; p = $4.2*10^{-8}$)
  - provided an independent replication of this SNP with caffeine^11^
- Caffeine-rs2597979 association remained after conditional analysis with rs10772420 (top quinine SNP, $r^2$ = 0.24)
  - explained 1.91% trait variance

:::
::: {.column}
![@Hwang2018](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12864-018-5058-2/MediaObjects/12864_2018_5058_Fig2_HTML.png) 

:::
::::::::::::::

## Functional Analysis
- 1.	PROP, quinine, caffeine, and SOA-associated SNPs include missense variants.
- 2.	PROP, quinine, and DB-associated SNPs are located within conserved region.
- 3.	Except for caffeine-associated SNPs, all SNPs are in high DNAse hypersensitivity sites.
- 4.	PROP, SOA, and DB-associated SNPs are inside promoter and/or enhancer regions.
- 5.	All SNPs may cause regulatory motif change of a protein. 
- 6.	All SNPs are eQTLs.
- 7.	Quinine, caffeine, and SOA-associated SNPs are mQTLs.
- 8.	None of these analyses was performed using taste tissues.

::: notes
(Is this important to focus on and understand?)
(from Table S11) (mTQL = methylation quantitative trait loci; eQTL = expression quantitative trait loci)
::: 

## What I can learn from this paper


## Appendix
![@Genick2011](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/Genick2011-Fig7.png)

# Questions

## References