---
title: "Journal Club Notes"
output: 
  html_document:
    keep-md: true
bibliography: MyLibrary.bib
---

# Objective:  
  1. present @HwangEtAl+shortyear as a journal club to the joint Simons-Cooperstone labs 

## Present:   
  1. Read the papers, *describe* the **general methodologies** used and **data** presented, 
  and *summarize* it and its **findings** 
  2. What do these findings *mean* for **your own** research?
  
  From Jess:  
  Hi Matthiew – I’d like to see you select one of the two papers I asked you to read and create a journal club like presentation describing what is done in the paper, and **highlighting how this work is relevant to your research**. I was you to understand the *basics of their methods* and the *implications of their findings*. You can focus on the coffee relevant parts of the paper to make it more manageable. You and Chris can think together about what a reasonable timeline to put this together would be.

I’m concerned there are some **fundamentals about genetic control of bitterness perception (including basics about what a gene is, what genetic variation really means, what a bitter receptor is)** as well as what **implication** this has for your project and how you attempt to control for this in your studies. I’d like to be sure this is something you are understanding moving forward.
  
# Outline:
## Intro/Background  
### 1. describe a problem  

- bitterness is a basic taste, but isn't perceived the same way for everyone  
- 36-73% of bitterness perception is due to genetics (@HwangEtAl+shortyear)  
  + *what are some explanations of variability outside of genetic reasons?*
  
### 2. describe a question  

- PROP bitterness has been implicated in fat perception (@tepperFatPerceptionRelated1997)  
- PTC related to diseases (@Guo_Reed_2001), immune system, and surgical outcome of severe rhinosinusitis (@HwangEtAl+shortyear)  
- want to understand "underlying genetic variation that leads to individual differences in bitter perception"
  + this could have implications in "personalized medicine" (@HwangEtAl+shortyear; @BushMoore+shortyear)

### 3. describe other's responses  

- Previous GWAS (@Reed2010) replicated association between *TAS2R38* and PROP perception  
  + and identified a SNP within *TAS2R19*: accounting for 5.8% of variance in quinine perception  
  + did not find loci for caffeine, SOA, or proposed second locus for PTC/PROP, suggested/proposed signals on chromosomes 5 and 16  
  
### 4. explain a need for a new/novel approach  

- use multivariate association analysis to find common genetic variants (MAF $\ge$ 5%) with small effects  
- can identify *pleiotropic* genetic variants
- can identify variants with only one of the correlated phenotypes
- using MV GWAS can find associations that didn't reach significance in univariate analysis

![**Figure 1**](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12864-018-5058-2/MediaObjects/12864_2018_5058_Fig1_HTML.png?as=webp)  

  + 1A: spread on y axis, not on x axis
    + only T1 (y axis) is associated with variant (non-pleiotropic)
    + better separation seen in 2D (practice with the pencil test from high school math class)
  + 1B: spread on y and x axis
    + minor allele has opposing effects, $\uparrow$ T1 (Y) and $\downarrow$ T2 (X)
    + starting to see the spread on y and x axes, especially when combined
  + 1C: minor allele has similar effects, $\uparrow$ T1 (Y) and $\uparrow$ T2 (X)
    + no greater separation with bivariate analysis than would get with univariate
    
### 5. say your aim  
![**Figure 1**](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e2a/3531285/2e868bea85f6/pcbi.1002822.g001.jpg)

- identify common genetic variants with small effect sizes (lower right of figure)
  + these variants may have been underpowered in the previous GWAS study (@Hansen2006)

## Methods  
### Taste Perception
- described in @Hansen2006  
- presented 5 bitter samples with reps
  - $2*10^{-4}\ M$ SOA
  - $1.81*10^{-4}\ M$ quinine HCl
  - $0.050 \ M$ caffeine
  - $6.0*10^{-4}\ M$ PROP
  - $4.99*10^{-6}\ M$ denatonium benzoate
  - same order presentation for all participants
- swish and spit after 5 s, rate on LMS
- bitter scores were square root transformed then converted to z-scores

### GWAS
- described in @HwangEtAl+shortyear
  - supplemental underpinings of GWAS from @BushMoore+shortyear
- 1999 Caucasians from Europe, including twins and siblings
  + this is an increase in sample size from @Hansen2006 by 40%
- used p < $5 * 10^{-8}$ for genome wide significance
  + used a Bonferroni corrected p < $1*10^{-8}$
  
  
#### Modeling
- linear mixed model
  + covariates (fixed effects): age, sex, ear infection history, first 5 PC from genotype
  + used additive model (cf. @BushMoore+shortyear)

  
- bivariate analysis can complement univariate analysis
  + can enhance the strength of a SNP association, but not effect size @HwangEtAl+shortyear

- variance explained by SNP was calculated as $2* MAF* (1-MAF)* \beta^2$

#### Functional annotation of identified SNPs
- functional annotation can be used to identify any potential roles of the identified SNPs
  + could be used to identify/develop mechanistic hypotheses in understanding variance in taste perception based on the identified/associates SNPs

## Results  
## Discussion
### Implications of *their* research
### Implications of *my* research
## Conclusions

# Notes
- read references 26 and 29 in Hwang et al

# References