---
title: "Journal Club Notes"
output: 
  html_document:
    keep-md: true
bibliography: MyLibrary.bib
---

# Objective:  
  1. present @Hwang2018 as a journal club to the joint Simons-Cooperstone labs 

## Present:   
  1. Read the papers, *describe* the **general methodologies** used and **data** presented, 
  and *summarize* it and its **findings** 
  2. What do these findings *mean* for **your own** research?
  
  From Jess:  
  Hi Matthiew – I’d like to see you select one of the two papers I asked you to read and create a journal club like presentation describing what is done in the paper, and **highlighting how this work is relevant to your research**. I was you to understand the *basics of their methods* and the *implications of their findings*. You can focus on the coffee relevant parts of the paper to make it more manageable. You and Chris can think together about what a reasonable timeline to put this together would be.

I’m concerned there are some **fundamentals about genetic control of bitterness perception (including basics about what a gene is, what genetic variation really means, what a bitter receptor is)** as well as what **implication** this has for your project and how you attempt to control for this in your studies. I’d like to be sure this is something you are understanding moving forward.
  
# Outline:
## PROP Case Study
![PROP signal, chromosome 7. Figure 1 from @Reed2010](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/hmg/19/21/10.1093_hmg_ddq324/1/ddq32401.jpeg?Expires=1741366531&Signature=BY8XsJBxFQKh~cPYTagz7iRR5w-pVch9GBHdvImEDK~Dn9GJlHU9o~UXzd1rT6VQNDanx1PWqKq09ubsyke9EDCVe1-rhsBCvTWrpOUk8ev0RiuRAsoY4KePIEqOrOrSP0R5yD1iBdjX1nLBqRtyC7PT3xpiC80YreDZYRVAm9NQ41bgoMCf4wElbu5IK3wkdbrxlDZrlmxORdbc7oZzWx6tuod1kP10cFftFhe9QGfj~th1ueT2R5wclWhk0f97t3QxN8Rafux8OXhWhsNfpnSll6PjxRcU1KJiu8KRjCfajzKOYJSUOrfxxVfLmkxK6VLo2-ylXEAjCGDDSZLygA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

- 3 SNPs (Robino 2022 [add to MyLibrary.bib])
  + PAV haplotype = "taster"
  + AVI haplotype = "nontaster"
- SNP rs10246939
  + explains ~50% (which) variance in detection thresholds
  + MAF 46% (@Genick2011)
    + Chris mentioned that because it is "ubiquitous", there is generally some associated advantage to this SNP as to why it wasn't slowly removed through natural selection

![Figure 4 from @Genick2011. Panels C and D how the relationship between PROP sensitivity and genotype at rs10246939](https://storage.googleapis.com/plos-corpus-prod/10.1371/journal.pone.0027745/1/pone.0027745.g004.PNG_L?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=wombat-sa%40plos-prod.iam.gserviceaccount.com%2F20250131%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250131T170127Z&X-Goog-Expires=86400&X-Goog-SignedHeaders=host&X-Goog-Signature=6944a8b0edf3d7315d43fe8765f40378785709602773f91deaf15215d3eb35cb68267241fabd71e0aff0835a534addc2ec8461806357f963601ee71eeb8aff336ce13a1b5051a282982b4d571a09aa531c057afe7c334230f5b581807626a9096635521754787276b152795bdcebeece4f91c955383ba4d048563150361aaa00247c3c8f9661d5a108c5bc0c09c31651948c270a37a0b3ea375d6b50fe07c14fece94fb4e706163d9ec9e54f6280c3127f20aac195825a130dfc3f8a96c19453aa36a03fd30199302f540d9ca52fbba3c5801eafb901d6849ee1adfc49a4d87566f2cd5bbcbca9d0194e4d294b8f20a94147a691833d865f98f3142a1e515e46)  

\

![Figure 7 from @Genick2011 showing the average perceived gLMS intensity ($\pm$) SEM for each genotype oat SNP rs10246939](https://storage.googleapis.com/plos-corpus-prod/10.1371/journal.pone.0027745/1/pone.0027745.g007.PNG_L?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=wombat-sa%40plos-prod.iam.gserviceaccount.com%2F20250131%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20250131T170921Z&X-Goog-Expires=86400&X-Goog-SignedHeaders=host&X-Goog-Signature=885ace5a7ae1c8666d61aae512dcdd4ced49a8ba73ba9389115df2176305a7e488b55c39f88dee6cd54c702a253eb303417b41100122b8fe81a6de72617dd46e9bc2ebf031b85b15d5bcaf2c3a01f30945b9b11973341f6b195e0365c4c3610588adb8603dc141c556ff359b2923d1854ac7d103b052c1bbb462155608cefe0ec9c754e8dcae177abf25a94379efbf470c10ed53827ff11d0d009e582291ea444388410a20cc9df638d28d7fca608ad822ea44e415748f6c0b3260bb8ac04263ba94047b9d074d1e17cdec88fa8a63f5d8b5d83a7a11f92068d2bdcde717140fdd51401f7088fd1f5a30774dfce7c9e228a91d81d25b740ebc90bf2228291add)  


- Previous GWAS (@Reed2010) showed signal at TAS2R38 (rs10246939)
  + also showed TAS2R19 for quinine on chromosome 12
  + NS for caffeine/SOA
  + no other proposed loci for PROP on chromosomes 5 or 16
  
## @Hwang2018
### Intro/Background  
#### 1. describe a problem  

- bitterness is a basic taste, but isn't perceived the same way for everyone  
- 36-73% of bitterness perception is due to genetics (@Hwang2018)  
  + *what are some explanations of variability outside of genetic reasons?*
  
#### 2. describe a question  

- PROP bitterness has been implicated in fat perception (@Tepper1997)  
- PTC related to diseases (@Guo2001), immune system, and surgical outcome of severe rhinosinusitis (@Hwang2018)  
- want to understand "underlying genetic variation that leads to individual differences in bitter perception"
  + this could have implications in "personalized medicine" (@Hwang2018; @Bush2012)

#### 3. describe other's responses  

- Previous GWAS (@Reed2010) replicated association between *TAS2R38* and PROP perception  
  + and identified a SNP within *TAS2R19*: accounting for 5.8% of variance in quinine perception  
  + did not find loci for caffeine, SOA, or proposed second locus for PTC/PROP, suggested/proposed signals on chromosomes 5 and 16  
  
#### 4. explain a need for a new/novel approach  

- use multivariate association analysis to find common genetic variants (MAF $\ge$ 5%) with small effects  
- can identify *pleiotropic* genetic variants
- can identify variants with only one of the correlated phenotypes
- using MV GWAS can find associations that didn't reach significance in univariate analysis

![**Figure 1**](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12864-018-5058-2/MediaObjects/12864_2018_5058_Fig1_HTML.png?as=webp)  

  + 1A: spread on y axis, not on x axis
    + only T1 (y axis) is associated with variant (non-pleiotropic)
    + better separation seen in 2D (practice with the pencil test from high school math class)
  + 1B: spread on y and x axis
    + minor allele has opposing effects, $\uparrow$ T1 (Y) and $\downarrow$ T2 (X)
    + starting to see the spread on y and x axes, especially when combined
  + 1C: minor allele has similar effects, $\uparrow$ T1 (Y) and $\uparrow$ T2 (X)
    + no greater separation with bivariate analysis than would get with univariate
    
#### 5. say your aim  
![**Figure 1**](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6e2a/3531285/2e868bea85f6/pcbi.1002822.g001.jpg)

- identify common genetic variants with small effect sizes (lower right of figure)
  + these variants may have been underpowered in the previous GWAS study (@Hansen2006)

### Methods  
#### Taste Perception
- described in @Hansen2006  
- presented 5 bitter samples with reps
  - $2*10^{-4}\ M$ SOA
  - $1.81*10^{-4}\ M$ quinine HCl
  - $0.050 \ M$ caffeine
  - $6.0*10^{-4}\ M$ PROP
  - $4.99*10^{-6}\ M$ denatonium benzoate
  - same order presentation for all participants
- swish and spit after 5 s, rate on LMS
- bitter scores were square root transformed then converted to z-scores

#### GWAS
- described in @Hwang2018
  - supplemental underpinings of GWAS from @Bush2012
- 1999 Caucasians from Europe, including twins and siblings
  + this is an increase in sample size from @Hansen2006 by 40%
- used p < $5 * 10^{-8}$ for genome wide significance
  + used a Bonferroni corrected p < $1*10^{-8}$
  
  
##### Modeling
- linear mixed model
  + covariates (fixed effects): age, sex, ear infection history, first 5 PC from genotype
  + used additive model (cf. @Bush2012)

  
- bivariate analysis can complement univariate analysis
  + can enhance the strength of a SNP association, but not effect size @Hwang2018

- variance explained by SNP was calculated as $2* MAF* (1-MAF)* \beta^2$

##### Functional annotation of identified SNPs
- functional annotation can be used to identify any potential roles of the identified SNPs
  + could be used to identify/develop mechanistic hypotheses in understanding variance in taste perception based on the identified/associates SNPs

### Results  
- *focus on PROP and caffeine* 

### Discussion

#### Implications of *their* research

#### Implications of *my* research

### Conclusions


# Notes
- read references 26 and 29 in Hwang et al

# References