---
title: "Reed et al (2010) Notes"
output: 
  html_document:
    keep-md: true
bibliography: MyLibrary.bib
---
# Abstract: 
![*Abstract of Reed et al (2010)*](C:/Users/Matthiew/Documents/Research/Candidacy/JournalClub/CandidacyJournalClub/Reed2010Abstract.png) 

# Intro
- PROP/PTC TAS2R38 explains most of the genetic variation, although other modifiers may exist
- @Hansen2006 found a common genetic factor accounting for 22-28% of the variance of caffeine, SOA, and quinine, and a quinine-specific factor accounts for 15% of the variation
  + this study only used modeling, not GWAS/genetics to identify what these factors/genes might be
- aim of this study is to use a GWAS to try to identify the specific genes responsible for the variation seen in @Hansen2006

# Methods
- used "discovery" and "replicate" samples
  + discovery sample was from Brisbane twin study
  + rep sample was collected from Ohio
- 1 mL sample was used and collected bitterness perception on LMS
  + 1mL is not very much (like the amount of saliva you'd swallow)
- linkage-based analysis for 
  + intenisty of quinine
    + no linkage seen at chromosome 12 (likely due to lack of power)
  + intensity of PROP in solution
  + intensity of PROP on paper
    + linkage seen near TAS2R38
- quinine and PROP (solution) did not appear normally distributed
- population stratification had trivial impact on results

# Results
- used genome wide significance of $1.136 * 10^{-8}$
  + caffeine and SOA did not have any significant SNPs
  + found no overlapping genetic associations between quinine, SOA, and caffeine 
    + @Delwiche2001 suggested these 3 compounds might have a related genetic factor
- PROP solution
  + TAS2R38 on chromosome 7 had strong signal
    + specifically *rs713598* and *rs10246939*
    + accounted for 45.9% of maximum trait variance
  + a modifier region TAS2R1 approached significance (*rs6867567*; chromosome 5; p $2.20 * 10*{-7}$)
- quinine
  + 19 SNPs were identified, mostly around bitter receptors on chromosome 12 and two salivary protein genes
  + one SNP coded fro an A-to-C sub at *rs10772420* R299C (amino acid #299) in bitter receptor TAS2R19 (chromosome 12)
    + A allele associated with more intense quinine perception
  + maximum trait variance of 5.77%
  
# Discussion
- quinine
  + study suggests proline-rich salivary proteins and bitter taste receptors
  + because of LD, the gene cannot be identified with GWAS alone
- PROP
  + supported that TAS2R38 is responsible for trait variance
  + a region on chromosome 5 approached significance
- adaptation to bitterness may be advantageous
  + sensitivity to bitterness: detect poinous plants/materials
  + decreased sensitivity to bitterness: consume nutritious plants that have toxins
- TAS2Rs are found elsewhere in the body (not limited to oral cavity)
  + nose, lungs, pancreas, GI tract

# Notes
- check out reference 40
- ![**Figure 3.** Average bitter taste intensity of quinine as rated by people from the discovery (top) and replication (bottom) samples grouped by their rs10772420 genotype. LMS, labeled magnitude scale; VAS, visual analogue scale. Values are means and confidence intervals corrected for family relationship (upper panel) or means and standard deviations (lower panel). The TAS2R19 A allele corresponds to the cysteine amino acid at position 299 and is associated with more intense perception of quinine in both samples.](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/hmg/19/21/10.1093_hmg_ddq324/1/ddq32403.jpeg?Expires=1741110950&Signature=OLyi-0l7QYlX7Q89Vw0kW5BstPsZanTA1Y920M5OQSKqvlXD3CZQU6bO~x6Ktc-gMwgn15gHfxC0IufVcT8WKgsDjm~Gi-SvBp2kaNsHk-VzGlFNKFdXS~TVsNORiTAnn9CNkwdcOr-FrnDzk-NeRi-hKMNVvkxn1aFmJcV8wTL2v9bg5Oke9E9N8Dqo3oWpU6iocf46o7e0kNxJ5MbyD3Yb6drNwe0lkaNjVXM-8EL4bezYmaAGkhDZ-3hZuvROI-JNojQjijKCM3AwZhlSy6wKOQfRNcUW~Bscda0geKjWKlzGMSV2baOT-OdSBw8VaJ9IxgkYZc2m~4XrKHDHJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)
  + a great example of what GWAS does
